README.md

```
@@ -1,3 +1,140 @@
+### 장기 판
+
+- [x] 장기판(JanggiBoard)
+  - 역할: 좌표와 기물 연결
+  - 좌표 Position
+  - [x] 행동: 다른 기물을 고려하여 이동 후보군 필터링
+
+- [x] 보드 초기화(BoardInitializer)
+  - 역할: 초기 장기 보드에 기물 배치
+  - [x] 행동: 초기화
+
+- [x] 경로(Route) 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)